marketplace-and-integrations/packages/authorized-services.md

```diff
@@ -0,0 +1,776 @@
+
+If the service is not yet authorized, click the _Authorize Service_ button to trigger the authentication and authorization flow. You will be directed to the service to login, and optionally choose an account.  You will then be asked to agree to the permissions requested by the app. Finally you will be redirected back to the Umbraco backoffice. You should see confirmation that an access token has been retrieved and stored such that the service is now authorized. If provided, you can click the _Verify Sample Request_ button to ensure that service's API can be called.
+
+### Calling an Service
+
+To make a call to an authorized service, you first need to obtain an instance of `IAuthorizedServiceCaller`. This is registered with the dependency injection framework. As such it can be injected into a controller, view or service class where it needs to be used.
+
+If making a request where all information is provided via the path and querystring, such as GET requests, the following method should be invoked:
+
+```csharp
+Task<TResponse> SendRequestAsync<TResponse>(string serviceAlias, string path, HttpMethod httpMethod);
+```
+
+The parameters for the request are as follows:
+
+- `serviceAlias` - the alias of the service being invoked (e.g. `github`).
+- `path` - the path to the API method being invoked (e.g. `/repos/Umbraco/Umbraco-CMS/contributors`).
+- `httpMethod` - the HTTP method to use for the request (e.g. `HttpMethod.Get`).
+
+There is also a type parameter:
+- `TResponse` - defines the strongly typed representation of the service method's response, that the raw response content will be deserialized into.
+
+If you need to provide data in the request an overload is available. This can be used for POST or PUT requests that trigger the creation or update of a resource:
+
+```csharp
+Task<TResponse> SendRequestAsync<TRequest, TResponse>(string serviceAlias, string path, HttpMethod httpMethod, TRequest? requestContent = null)
+    where TRequest : class;
+```
+
+The additional parameter is:
+
+- `requestContent` - the strongly typed request content, which will be serialized and provided in the request.
+
+And additional type parameter:
+- `TRequest` - defines the strongly typed representation of the request content.
+
+
+If you need to work with the raw JSON response, there are equivalent methods for both of these that omit the deserialization step:
+
+```csharp
+Task<string> SendRequestRawAsync(string serviceAlias, string path, HttpMethod httpMethod);
+
+Task<string> SendRequestRawAsync<TRequest>(string serviceAlias, string path, HttpMethod httpMethod, TRequest? requestContent = null)
+    where TRequest : class;
+```
+
+Finally, there are convenience extension methods available for each of the common HTTP verbs. These allow you to simplify the requests and omit the `HttpMethod` parameter, e.g.
+
+```csharp
+Task<TResponse> GetRequestAsync<TResponse>(string serviceAlias, string path);
+```
+
+## Verified Providers
```

Comment:
How about we add these into "expandables" when on GitBook?
Or perhaps tabs? 🤔 
There might be too many different once for using tabs - especially if you're planning on adding even more.
To expendables might be the best option.

Let me know what you think!

Reply:
Sounds good too - I've moved these into expandable panels.